### PR TITLE
Be big, be bold

### PR DESCRIPTION
**Pull request by palewire**

### CODE REVIEW

**Comment by choldgraf** [comment]
What do you think instead about just un-setting this? I've read that [it is more accessible to not set a default font size for copy and instead use the system default](https://css-tricks.com/accessible-font-sizing-explained/#aa-avoid-setting-a-base-font-size)

**Comment by palewire** [comment]
I'm a news guy. So I like big. ;)

File: _sass/components/_copy.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 p {
   font-family: $font-family-sans-serif;
-  font-size: 16px;
+  font-size: 18px;
```
Comment (choldgraf) [inline _sass/components/_copy.scss:5]:
```suggestion
  // Unset the font size to let the system choose it, for a11y purposes
  // ref: https://css-tricks.com/accessible-font-sizing-explained/#aa-avoid-setting-a-base-font-size
  font-size: unset;
```

**Comment by choldgraf** [comment]
OK I just added a commit to try this, let's see how the end result looks. When I did this with Jupyter Book, it made all the fonts slightly larger anyway :-)

File: _sass/components/_copy.scss [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 p {
   font-family: $font-family-sans-serif;
-  font-size: 16px;
+  // Unset the font size to let the system choose it, for a11y purposes
+  // ref: https://css-tricks.com/accessible-font-sizing-explained/#aa-avoid-setting-a-base-font-size
+  font-size: unset;
```
Comment (choldgraf) [inline _sass/components/_copy.scss:7]:
```suggestion
  // Increase font size for easier reading + a11y
  font-size: 18px;
```

**Comment by choldgraf** [comment]
eh - the preview after unsetting actually shrank it, I reverted my change, let's look again

**Review by choldgraf — APPROVED** [review]
I think the bigger font makes it easier to read, likely more accessible as well, so I'm +1

**Pull request merged**